core/src/main/java/org/lflang/generator/c/CCmakeGenerator.java

```diff
@@ -519,4 +540,134 @@ private static String setUpMainTargetRp2040(
     code.newLine();
     return code.toString();
   }
-}
+
+
+  private static String setUpMainTargetStm32(
+          boolean hasMain, String executableName, Stream<String> cSources, String[] boardProperties) {
+    var code = new CodeBuilder();
+    code.pr("\n# ########################################################");
```

Comment:
I think we should use consistent formatting on the comments generated in CMakeLists.txt